client/my-sites/current-site/index.jsx

```diff
@@ -110,15 +111,17 @@ class CurrentSite extends Component {
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		const cardClasses = classNames( 'current-site', {
+			multisite: this.props.siteCount > 1,
```

Comment:
What's the purpose of this new class? It doesn't seem to be defined anywhere else 🤔